### PR TITLE
Cart component

### PR DESCRIPTION
**Pull request by malaykhakhar**

### CODE REVIEW

File: style/cart.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+}
+
+.separator {
+  margin: auto;
+  width: 90%;
+  background-color: #d3d3d3;
```
Comment (parvez-rashid) [inline style/cart.css:54]: You can use variables here.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  margin: 0;
+  font-size: 1.6rem;
+  font-family: "Inter", sans-serif;
+}
+
+.nav-search {
+  height: 3.5rem;
+  width: 20rem;
+  border-radius: 10px;
+  border: 1px solid #bcbcbc;
```
Comment (parvez-rashid) [inline style.css:34]: You can use variables here.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  font-size: 62.5%;
+}
+
+.pos-rel {
+  position: relative;
+}
+
+.pos-abs {
+  position: absolute;
+}
```
Comment (parvez-rashid) [inline style.css:13]: Great that you've used utility classes.

**Review by parvez-rashid — APPROVED** [review]
UI Review: UI looks great, just one suggestion, try to use a different cart icon, the current one is a bit unclear whether it's a bag or a lock. Try to use an icon similar in weight to the wishlist icon.

Code looks great, have a few comments below. Other than that, can be merged.

File: pages/cart.html [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+          <ul>
+            <li>
+              <input type="text" placeholder="Search" class="nav-search" />
+            </li>
+            <li>
+              <a href="#"><img src="/assets/images/wishlist.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="/assets/images/Cart-nav.svg" alt="" /></a>
+            </li>
```
Comment (Sahil25061999) [inline pages/cart.html:24]: give alt name

File: style/cart.css [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  height: 1px;
+  margin-top: 10px;
+  margin-bottom: 20px;
```
Comment (Sahil25061999) [inline style/cart.css:57]: use rem here

**Review by Sahil25061999 — APPROVED** [review]
Most of the code looks good to me, just a ui suggestion give secondary button styling to remove cart button this will lead to better user experience. Love the primary color used for the site.

**Pull request merged**